workflows/src/test/kotlin/com/r3/developers/apples/workflows/ApplesFlowDriverTests.kt

```diff
@@ -42,22 +40,22 @@ class ApplesFlowDriverTests {
     @RegisterExtension
     private val driver = DriverNodes(alice, bob).withNotary(notary, 1).forAllTests()
 
+    private lateinit var vNodes: Map<MemberX500Name, VirtualNodeInfo>
+
     @BeforeAll
     fun setup() {
-        driver.run { dsl ->
+        vNodes = driver.let { dsl ->
             dsl.startNodes(setOf(alice, bob))
-                .filter { it.cpiIdentifier.name == "workflows" }
-                .associateByTo(vNodes) { it.holdingIdentity.x500Name }
+            dsl.nodesFor("workflows")
         }
-        if (vNodes.isEmpty()) Assertions.fail<Any>("Failed to populate vNodes")
+        assertThat(vNodes).withFailMessage("Failed to populate vNodes").isNotEmpty()
```

Comment:
this would read better if it was 
```
assertThat(vNodes).isNotEmpty().withFailMessage("Failed to populate vNodes")
```

Reply:
Maybe, but it wouldn't work in the same way:
```kotlin
assertThat(emptyMap<MemberX500Name, VirtualNodeInfo>())
    .isNotEmpty()
    .withFailMessage("Failed to populate vNodes")
```
generates:
```
java.lang.AssertionError: 
Expecting actual not to be empty
	at com.r3.developers.apples.workflows.ApplesFlowDriverTests.setup(ApplesFlowDriverTests.kt:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        ...
```
whereas
```kotlin
assertThat(emptyMap<MemberX500Name, VirtualNodeInfo>())
    .withFailMessage("Failed to populate vNodes")
    .isNotEmpty()
```
generates
```
java.lang.AssertionError: Failed to populate vNodes
	at com.r3.developers.apples.workflows.ApplesFlowDriverTests.setup(ApplesFlowDriverTests.kt:51)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        ...
```

Reply:
Fair enough, must be the way the builder underneath handles it